src/UuidFactory.php

```diff
@@ -180,6 +187,24 @@ public function setUuidBuilder(UuidBuilderInterface $builder)
         $this->uuidBuilder = $builder;
     }
 
+    /**
+     * @return ValidatorInterface
+     */
+    public function getValidator()
+    {
+        return $this->validator;
+    }
+
+    /**
+     * @return void
+     */
+    public function setValidator(ValidatorInterface $validator)
+    {
+        assert(func_num_args() === 1);
```

Comment:
True, I guess it just depends on how strong you like method signatures to be. I'll remove this since it's just a library though